### DISCUSSION TITLE
Lossless Cut - My exported files end up as unusable .bin or .csv files NOT video files?

### DISCUSSION THREAD

**Discussion by coffee-king** [mifi/lossless-cut#799] [Q&A]
Hi Guys
Something seems to be wrong with Lossless Cut on my Mac as my exported files end up as unusable .bin or .csv files NOT video files. I've tried a reinstall and thats not done anything. Any help anyone please?
Thanks in advance.

**Comment by mifi** [comment]
Please provide a sample video or some more details such as screenshots of how to reproduce the problem. and maybe a report (help menu)

**Reply by coffee-king** [reply]
Hi
Oddly I'm now having problems replicating this .bin exported file issue, but it's doing something else instead now.
I'm using (And always was using) footage from a Sony a7Siii. When I try to export as a .mov file with Lossless Cut it wont play back in VLC Player as it states "Could not decode the format AIVX". When I try to open in Quick Time it just shows a green coloured screen instead of the footage. It wont export as MP4, but will export as MKV. But as I can't import MKV files into Final Cut Pro I'd then have to run it through ANOTHER converter which seems a waste of time.
Any thoughts please?
(If I can replicate .bin issue again I will post it on here)

**Comment by mifi** [comment]
it's hard to say anything without any sample file or report from the help menu

**Reply by coffee-king** [reply]
Hi
Can you advise how I do a Report please?

**Reply by mifi** [reply]
open your file, then: <img width="357" alt="Screenshot 2021-08-24 at 17 34 55" src="https://user-images.githubusercontent.com/402547/130602250-7347f35d-db67-4924-a355-24886afd9a9c.png">

**Reply by faxedick** [reply]
Hi, I have same problem at Windows 10 PC's. Used newest version and also tried some other versions like 3.21. Same problem. My cutted files get stored without any file extension. So it looks like "Filename." "mp4" is missing. When I rename the file in "Filename.mp4" it works.
Also have had sometimes .csv as fileextension. Tried this on two PC with same problem. I used before a very old version of Lossless cut and this still works.

**Reply by mifi** [reply]
please provide a more detailed report and how to reproduce the problem. i cannot work with this. see the issues template

**Reply by faxedick** [reply]
Thanks for fast reply!
This one?
No error

{
  "state": {
    "filePath": "D:\\Videos\\Aruma Video 2\\ausschuss\\VID_20201108_141648.mp4",
    "fileFormat": "mp4",
    "externalStreamFiles": [],
    "mainStreams": [
      {
        "index": 0,
        "codec_name": "h264",
        "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10",
        "profile": "High",
        "codec_type": "video",
        "codec_tag_string": "avc1",
        "codec_tag": "0x31637661",
        "width": 1920,
        "height": 1080,
        "coded_width": 1920,
        "coded_height": 1080,
        "closed_captions": 0,
        "has_b_frames": 0,
        "sample_aspect_ratio": "1:1",
        "display_aspect_ratio": "16:9",
        "pix_fmt": "yuv420p",
        "level": 40,
        "color_range": "tv",
        "color_space": "bt709",
        "color_transfer": "bt709",
        "color_primaries": "bt709",
        "chroma_location": "left",
        "refs": 1,
        "is_avc": "true",
        "nal_length_size": "4",
        "r_frame_rate": "179/6",
        "avg_frame_rate": "14670000/518143",
        "time_base": "1/90000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 518137,
        "duration": "5.757078",
        "bit_rate": "9927763",
        "bits_per_raw_sample": "8",
        "nb_frames": "163",
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0
        },
        "tags": {
          "creation_time": "2020-11-08T13:16:48.000000Z",
          "language": "eng",
          "handler_name": "VideoHandle",
          "vendor_id": "[0][0][0][0]"
        }
      },
      {
        "index": 1,
        "codec_name": "aac",
        "codec_long_name": "AAC (Advanced Audio Coding)",
        "profile": "LC",
        "codec_type": "audio",
        "codec_tag_string": "mp4a",
        "codec_tag": "0x6134706d",
        "sample_fmt": "fltp",
        "sample_rate": "48000",
        "channels": 2,
        "channel_layout": "stereo",
        "bits_per_sample": 0,
        "r_frame_rate": "0/0",
        "avg_frame_rate": "0/0",
        "time_base": "1/48000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 276480,
        "duration": "5.760000",
        "bit_rate": "192001",
        "nb_frames": "270",
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0
        },
        "tags": {
          "creation_time": "2020-11-08T13:16:48.000000Z",
          "language": "eng",
          "handler_name": "SoundHandle",
          "vendor_id": "[0][0][0][0]"
        }
      }
    ],
    "copyStreamIdsByFile": {
      "D:\\Videos\\Aruma Video 2\\ausschuss\\VID_20201108_141648.mp4": {
        "0": true,
        "1": true
      }
    },
    "cutSegments": [
      {
        "start": 1.0949170415814589,
        "end": 3.390710838445808
      }
    ],
    "fileFormatData": {
      "filename": "D:\\Videos\\Aruma Video 2\\ausschuss\\VID_20201108_141648.mp4",
      "nb_streams": 2,
      "nb_programs": 0,
      "format_name": "mov,mp4,m4a,3gp,3g2,mj2",
      "format_long_name": "QuickTime / MOV",
      "start_time": "0.000000",
      "duration": "5.760000",
      "size": "8092884",
      "bit_rate": "11240116",
      "probe_score": 100,
      "tags": {
        "major_brand": "mp42",
        "minor_version": "0",
        "compatible_brands": "isommp42",
        "creation_time": "2020-11-08T13:16:48.000000Z",
        "com.android.version": "10"
      }
    },
    "rotation": 360,
    "shortestFlag": false
  },
  "platform": "win32",
  "version": "3.42.0"
}

**Reply by mifi** [reply]
Thanks. what does the output format button say?
<img width="170" alt="Screenshot 2022-01-18 at 23 52 14" src="https://user-images.githubusercontent.com/402547/149982014-064eb4a3-4884-4e2e-a2ac-fa59cb85fb13.png">

and have you set any custom "Output name(s):"?

**Reply by faxedick** [reply]
OMG, ok that was the problem..... I changed the output name and did not know that you have to write the fileextension too. Thougt .mp4 will be added by program!

Thank you for fast support!
